packages/shared/src/utils/General.ts

```
@@ -74,22 +78,91 @@ export function convertWiFiObjToString(wiFiInput: TWifiInput): string {
+	const encodedBody = body ? encodeURIComponent(body) : '';
+	return `mailto:${email}?subject=${encodedSubject}&body=${encodedBody}`;
+};
+
+export const convertQRCodeDataToStringByType = (
+	content: TQrCodeContent,
+	shortUrl?: string,
+): string => {
 	switch (content.type) {
 		case 'url':
-			const { url, isEditable, shortUrl } = content.data as unknown as any;
+			const { url, isEditable } = content.data as unknown as any;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)